lib/glific/contacts/import.ex

```diff
@@ -19,30 +19,33 @@ defmodule Glific.Contacts.Import do
 
   @max_concurrency System.schedulers_online()
 
-  @spec cleanup_contact_data(map(), non_neg_integer, String.t()) :: map()
-  defp cleanup_contact_data(data, organization_id, date_format) do
-    %{
-      name: data["name"],
-      phone: data["phone"],
-      organization_id: organization_id,
-      language_id: Enum.at(Settings.get_language_by_label_or_locale(data["language"]), 0).id,
-      optin_time:
-        if(data["opt_in"] != "",
-          do: elem(Timex.parse(data["opt_in"], date_format), 1),
-          else: nil
-        ),
-      delete: data["delete"],
-      contact_fields: Map.drop(data, ["phone", "language", "opt_in", "delete"])
-    }
-  end
-
-  @spec add_contact_to_group(Contact.t(), non_neg_integer) :: {:ok, ContactGroup.t()}
-  defp add_contact_to_group(contact, group_id) do
-    Groups.create_contact_group(%{
-      contact_id: contact.id,
-      group_id: group_id,
-      organization_id: contact.organization_id
-    })
+  @spec cleanup_contact_data(map(), map(), String.t()) :: map()
+  defp cleanup_contact_data(data, %{user: user, organization_id: organization_id}, date_format) do
+    if user == [:Glific_admin] do
```

Comment:
Also user === [:Glific_admin] does not make sense here. Do you mean user_role == [:Glific_admin]  ?